src/pivotal-ui/components/dropdowns/dropdowns.scss

```diff
@@ -159,7 +159,7 @@ var Dropdown = require('pui-react-dropdowns').Dropdown;
 This is the basic bootstrap dropdown:
 
 ```react_example_table
-<UI.Dropdown title='DropDown'>
+<UI.Dropdown title='DropDown' buttonClassName="my-btn-class">
```

Comment:
Two problems with this documentation

1) We should explain that buttonClassName is optional. It's confusing in this example - it looks like you need to include the buttonClassName property in order for the component to work.

2) Instead of a made-up class name, we should show an example with a className that creates an obvious appearance difference in the Dropdown - like `btn-large` (or whatever it is).